professionalism/professionalism.md

```diff
@@ -19,6 +19,15 @@ About 2 hours
 ### Motivation
 Techtonica is a simulated work environment representative of common tech industry offices. Apprentices are expected to behave professionally and to interact professionally with their Techtonica colleagues and staff. An apprentice's level of professionalism is an important point of consideration for sponsoring companies.
 
+Importance to professionalism in workplace:
+- Creates Boundaries – Workers who conduct themselves professionally avoid crossing their line with subordinates, superiors or clients
```

Comment:
These are great, but this is getting quite long. Can you make a new subsection for all this at the beginning of Lesson, and just list the topics here? Like:
Importance of..., :
- creates boundaries 
- creates an atmosphere of improvement 
Etc